From 88b853275836df6a4ff3965383dc0a0648550922 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Wed, 21 Jun 2017 10:57:33 +0800
Subject: [PATCH 1978/5242] MLK-15131-2 ARM64: dts: freescale: imx8mq: enable
 snvs rtc

commit  b9e870da1f4be7c9e16d7ebd3993bd02ba82fc8c from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable SNVS RTC by default on i.MX8MQ.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 8667a6d..5f64dcc 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -243,7 +243,6 @@
 			offset = <0x34>;
 			interrupts = <GIC_SPI 19 IRQ_TYPE_LEVEL_HIGH>,
 				<GIC_SPI 20 IRQ_TYPE_LEVEL_HIGH>;
-			status = "disabled";
 		};
 	};
 
-- 
1.7.9.5

